jupyterlab_gcedetails/src/components/hardware_scaling_form.tsx

```diff
@@ -114,16 +106,11 @@ export class HardwareScalingForm extends React.Component<Props, State> {
     });
   }
 
-  private onBaseMachineTypeChange(event: React.ChangeEvent<HTMLSelectElement>) {
-    this.setState({
-      baseMachineType: event.target.value,
-      cpuValue: this.getMachineType(event.target.value).configurations[0].cpu,
-      memoryValue: this.getMachineType(event.target.value).configurations[0]
-        .memory,
-    });
+  private onMachineTypeChange(machineType: Option) {
```

Comment:
Could just make this an inline handler now that the rest of the logic has been removed.

Reply:
Good point!